frontend/src/lib/components/Users/UserProjects.svelte

```diff
@@ -0,0 +1,59 @@
+<script  context="module" lang="ts">
+  // We define the Project type that we'll want in here, and export it so that callers can know they're passing in the right type
+  export type Project = {
+    id: string
+    name: string
+    code: string
+    role: ProjectRole
```

Comment:
this is the user role right? could we make that more explicit with a different name?

Reply:
Yes, the role of the user whose projects we're listing.

Reply:
Done in commit e81f084b.